crates/compiler/gen_llvm/src/llvm/lowlevel.rs

```diff
@@ -1409,6 +1409,15 @@ pub(crate) fn run_low_level<'a, 'ctx>(
 
             call_bitcode_fn(env, &[], bitcode::UTILS_DICT_PSEUDO_SEED)
         }
+        CryptEmptySha256 => call_bitcode_fn(env, &[], bitcode::CRYPT_EMPTY_SHA256),
+        CryptSha256AddBytes => {
+            arguments!(sha, data);
+            call_bitcode_fn(env, &[sha, data], bitcode::CRYPT_SHA256_ADD_BYTES)
+        }
+        CryptSha256Digest => {
+            arguments!(sha);
+            call_bitcode_fn(env, &[sha], bitcode::CRYPT_SHA256_DIGEST)
```

Comment:
I think you need to use `call_list_bitcode_fn` and put the list as the first argument to the function. I think that should fix the current bug.